sdk/storage/AzureStorageBlob/Source/StorageBlobClient.swift

```diff
@@ -84,16 +84,16 @@ public final class StorageBlobClient: PipelineClient {
     ///   - authPolicy: An `Authenticating` policy to use for authenticating client requests.
     ///   - options: Options used to configure the client.
     private init(
-        baseUrl: URL,
+        endpoint: URL,
         authPolicy: Authenticating,
         withOptions options: StorageBlobClientOptions
     ) throws {
         self.options = options
         super.init(
-            baseUrl: baseUrl,
-            transport: URLSessionTransport(),
-            policies: [
-                UserAgentPolicy(for: StorageBlobClient.self, telemetryOptions: self.options.telemetryOptions),
+            endpoint: endpoint,
+            transport: options.transportOptions.transport ?? URLSessionTransport(),
+            policies: options.transportOptions.perRequestPolicies ?? [
```

Comment:
Now that we merge policies I think this needs to be reverted

Reply:
Yes, it does.